app/src/main/java/com/example/parkingsystemkotlin/mvp/model/reservation/ReservationInformationDB.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.parkingsystemkotlin.mvp.model.reservation
+
+object ReservationInformationDB : DataBase {
+
+    private var reservationInfomation: MutableMap<String, MutableList<Reservation>>
+
+    init {
+        reservationInfomation = mutableMapOf()
+    }
+
+    override fun getReservationsDB(place: String): MutableList<Reservation>? = reservationInfomation[place]
```

Comment:
Why return a **MutableList<Reservation>?** ?? instead of saving a null, you can save a empty list

Reply:
done